gradle/itest.gradle

```diff
@@ -2,6 +2,8 @@
 // Integration tests support
 //
 
+
+// FIXME: this is not necessary anymore with new version of Idea
```

Comment:
If not necessary, should we remove all occurrences of this code in Gradle code ?